src/typesetting/maskingProcedure.md

```diff
@@ -0,0 +1,126 @@
+# Masking using AI and PS
+## Procedure
+
+Before we start, note that we will be
+doing all the grunt work in PS while keeping
```

Comment:
Breaking a line between "will be" and "doing" makes no sense. There are a couple other instances with questionable line breaks, but this stood out the most to me.

Please try to rebreak the lines at more structural berak points within a sentence, such as between subject, predicate, object, or around interjections and similar. Kind of how you would break long code lines between parameters, chained function calls, or around binary operators.

Reply:
This is something I cannot do since I am not from English background and not good enough in it to do it myself. If someone do it I would appreciate it

Reply:
Fine with me, I'll just do a quick pass on some of the sentences myself.